core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -15,312 +15,327 @@
-  public static final String WORKER_NETWORK_NETTY_SHUTDOWN_QUIET_PERIOD =
-      "alluxio.worker.network.netty.shutdown.quiet.period";
-  public static final String WORKER_NETWORK_NETTY_SHUTDOWN_TIMEOUT =
-      "alluxio.worker.network.netty.shutdown.timeout";
-  public static final String WORKER_ALLOCATOR_CLASS = "alluxio.worker.allocator.class";
-  public static final String WORKER_EVICTOR_CLASS = "alluxio.worker.evictor.class";
-  public static final String WORKER_EVICTOR_LRFU_STEP_FACTOR =
-      "alluxio.worker.evictor.lrfu.step.factor";
-  public static final String WORKER_EVICTOR_LRFU_ATTENUATION_FACTOR =
-      "alluxio.worker.evictor.lrfu.attenuation.factor";
-  public static final String WORKER_TIERED_STORE_LEVELS = "alluxio.worker.tieredstore.levels";
-  public static final String WORKER_FILESYSTEM_HEARTBEAT_INTERVAL_MS =
-      "alluxio.worker.filesystem.heartbeat.interval.ms";
-  public static final String WORKER_FILE_PERSIST_POOL_SIZE =
-      "alluxio.worker.file.persist.pool.size";
-  public static final String WORKER_FILE_PERSIST_RATE_LIMIT_ENABLED =
-      "alluxio.worker.file.persist.rate.limit.enabled";
-  public static final String WORKER_FILE_PERSIST_RATE_LIMIT =
-      "alluxio.worker.file.persist.rate.limit";
-  public static final String WORKER_TIERED_STORE_BLOCK_LOCKS =
-      "alluxio.worker.tieredstore.block.locks";
   /**
-   * This format is used as a template to generate the property name for a given level. e.g.,
-   * {@code String.format(Constants.WORKER_TIERED_STORE_LEVEL_ALIAS_FORMAT, level)}
+   * Passed to fuse-mount, maximum granularity of write operations:
+   * Capped by the kernel to 128KB max (as of Linux 3.16.0),.
    */
+  FUSE_MAXWRITE_BYTES("alluxio.fuse.maxwrite.bytes"),
+  /** Have the fuse process log every FS request. */
+  FUSE_DEBUG_ENABLE("alluxio.fuse.debug.enabled"),
+  /** Maximum number of Alluxio paths to cache for fuse conversion. */
+  FUSE_CACHED_PATHS_MAX("alluxio.fuse.cached.paths.max"),
+  FUSE_DEFAULT_MOUNTPOINT("alluxio.fuse.mount.default"),
+  FUSE_FS_ROOT("alluxio.fuse.fs.root"),
+  /** FUSE file system name. */
+  FUSE_FS_NAME("alluxio.fuse.fs.name"),
+  OSS_ACCESS_KEY("fs.oss.accessKeyId"),
+  OSS_SECRET_KEY("fs.oss.accessKeySecret"),
+  OSS_ENDPOINT_KEY("fs.oss.endpoint"),
+  S3N_ACCESS_KEY("fs.s3n.awsAccessKeyId"),
+  S3N_SECRET_KEY("fs.s3n.awsSecretAccessKey"),
+  // Not prefixed with fs, the s3a property names mirror the aws-sdk property names for ease of use
+  S3A_ACCESS_KEY("aws.accessKeyId"),
+  S3A_SECRET_KEY("aws.secretKey"),
+  GCS_ACCESS_KEY("fs.gcs.accessKeyId"),
+  GCS_SECRET_KEY("fs.gcs.secretAccessKey"),
+  SWIFT_USER_KEY("fs.swift.user"),
+  SWIFT_TENANT_KEY("fs.swift.tenant"),
+  SWIFT_API_KEY("fs.swift.apikey"),
+  SWIFT_PASSWORD_KEY("fs.swift.password"),
+  SWIFT_AUTH_URL_KEY("fs.swift.auth.url"),
+  SWIFT_AUTH_PORT_KEY("fs.swift.auth.port"),
+  SWIFT_AUTH_METHOD_KEY("fs.swift.auth.method"),
+  SWIFT_USE_PUBLIC_URI_KEY("fs.swift.use.public.url"),
```

Comment:
I didn't update them as their string property name is anyway different from our alluxio properties. and the naming is inconsistent anyway. probably we need separate discussion on the naming convention on those properties.